meeting_minutes/2022-08-30.md

```diff
@@ -0,0 +1,19 @@
+# working group meeting on 2022-08-30
+
+The agenda is in the [Kanban board](https://github.com/llvm/llvm-iwg/projects/1)
+
+## Participants
+
+* Luke Nihlen
+* Konrad Kleine
+* Kamau Bridgeman
+* Mehdi Amini
+* Anton Korobeynikov
+
+## Upcoming Discussion Between Board and IWG
+
+* Pre-Merge testing
+ * Could use an opt-in process followed by a proposal
+* Decision process? Who makes infra decisions for LLVM?
```

Comment:
Mentioned this: https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md
This is the process that was used to add Bazel infra support to LLVM/MLIR